src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -111,6 +111,7 @@ object FileMetadataService {
                   fileName: Option[String] = None,
                   parentId: Option[UUID] = None,
                   metadata: FileMetadataValues,
+                  filestatus: Option[String] = None,
```

Comment:
Just a small thing but can we have this as `fileStatus` instead of `filestatus` We're using camel case in other places for multiple words.